src/Features/Core/Portable/Completion/SharedSyntaxContextsWithSpeculativeModel.cs

```diff
@@ -5,47 +5,43 @@
 using System;
 using System.Collections.Concurrent;
 using System.Collections.Immutable;
+using System.Diagnostics;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.Shared.Extensions.ContextQuery;
-using Roslyn.Utilities;
 
-namespace Microsoft.CodeAnalysis.Completion
+namespace Microsoft.CodeAnalysis.Completion;
+
+internal sealed class SharedSyntaxContextsWithSpeculativeModel
 {
-    internal sealed class SharedSyntaxContextsWithSpeculativeModel
-    {
-        private readonly Document _document;
-        private readonly int _position;
+    private readonly Document _document;
+    private readonly int _position;
 
-        private readonly ConcurrentDictionary<Document, AsyncLazy<SyntaxContext>> _cache;
-        private readonly Lazy<ImmutableArray<DocumentId>> _lazyRelatedDocumentIds;
+    private readonly ConcurrentDictionary<Document, Task<SyntaxContext>> _cache;
```

Comment:
Change in this file just replaces the usage of AsyncLazy with a Task, unrelated to other changes